tests/test_modeling_xlm_roberta_xl.py

```diff
@@ -0,0 +1,61 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+from transformers import is_torch_available
+from transformers.testing_utils import require_sentencepiece, require_tokenizers, require_torch, slow
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import XLMRobertaXLModel
```

Comment:
We need more tests here. We can more or less copy all the tests from: https://github.com/huggingface/transformers/blob/e68c3756fea7c811d02b8470539ae17ec3ec0e71/tests/test_modeling_roberta.py#L49

Reply:
Thank you for the review, but the original xlm-roberta also doesn't have full tests like roberta. And I have followed the xlm-roberta test cases.